modules/customizing-the-jenkins-image-stream-tag.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: PROCEDURE
+
+[id="customizing-the-jenkins-image-stream-tag_{context}"]
+= Customizing the Jenkins image stream tag
+
+To control how the Jenkins image is upgraded, you can override the default upgrade behavior by setting the image stream tag value that your Jenkins deployment configurations use.
+
+The default upgrade behavior is the behavior that existed when the Jenkins image was part of the install payload. The image stream tag names, `2` and `ocp-upgrade-redeploy`, in the `jenkins-rhel.json` image stream file use SHA-specific image references. Therefore, when those tags are updated with a new SHA, the {product-title} image change controller automatically redeploys the Jenkins deployment configuration from the associated templates, such as `jenkins-ephermeral.json` or `jenkins-persistent.json`.
+
+For new deployments, to override that default value, you change the value of the  `JENKINS_IMAGE_STREAM_TAG` in the `jenkins-ephemeral.json` Jenkins template. For example, you replace the `2` in `"value": "jenkins:2"` with one of the following image stream tags:
```

Comment:
nit: should we use impersonal phrasing? (passive form, I forgot if this is the style standard or not)

Reply:
Red Hat prefers second person (you) and active voice.